arch/csr/hstateen0.yaml

```diff
@@ -0,0 +1,100 @@
+# yaml-language-server: $schema=../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: hstateen0
+long_name: Hypervisor State Enable 0 Register
+address: 0x60C
+priv_mode: S
+length: 64
+description: |
+  For each hstateen CSR, bit 63 is defined to control access to the matching sstateen CSR.
+  Bit 63 of hstateen0 controls access to sstateen0.
+
+  With the hypervisor extension, the hstateen CSRs have identical encodings to the mstateen
+  CSRs, except controlling accesses for a virtual machine (from VS and VU modes).
+
+  For every bit in an hstateen CSR that is zero (whether read-only zero or set to zero),
+  the same bit appears as read-only zero in sstateen when accessed in VS-mode.
+
+  A bit in an hstateen CSR cannot be read-only one unless the same bit is read-only one
+  in the matching mstateen CSR.
+
+definedBy:
+  allOf:
+    - H
+    - Smstateen
+    - Ssstateen
+fields:
+  SEO:
+    location: 63
+    description: |
+      The SE0 bit in hstateen0 controls access to the sstateen0 CSR.
+    type: RW
+    reset_value: UNDEFINED_LEGAL
+  ENVCFG:
+    location: 62
+    description: |
+      The ENVCFG bit in hstateen0 controls access to the senvcfg CSRs.
+    type: RW
+    reset_value: UNDEFINED_LEGAL
+  CSRIND:
+    location: 60
+    description: |
+      The CSRIND bit in hstateen0 controls access to the siselect and the
+      sireg*, (really vsiselect and vsireg*) CSRs provided by the Sscsrind
+      extensions.
+    type: RW
+    reset_value: UNDEFINED_LEGAL
+  AIA:
+    location: 59
+    description: |
+      The AIA bit in hstateen0 controls access to all state introduced by
+      the Ssaia extension and is not controlled by either the CSRIND or the
+      IMSIC bits of hstateen0.
+    type: RW
+    reset_value: UNDEFINED_LEGAL
+  IMSIC:
+    location: 58
+    description: |
+      The IMSIC bit in hstateen0 controls access to the guest IMSIC state,
+      including CSRs stopei (really vstopei), provided by the Ssaia extension.
+
+      Setting the IMSIC bit in hstateen0 to zero prevents a virtual machine
+      from accessing the hart’s IMSIC the same as setting `hstatus.`VGEIN = 0.
+    type: RW
+    reset_value: UNDEFINED_LEGAL
+  CONTEXT:
+    location: 57
+    description: |
+      The CONTEXT bit in hstateen0 controls access to the scontext CSR provided
+      by the Sdtrig extension.
+    type: RW
+    reset_value: UNDEFINED_LEGAL
+  JVT:
+    location: 2
+    description: |
+      The JVT bit controls access to the jvt CSR provided by the Zcmt extension.
+    type: RW
+    reset_value: UNDEFINED_LEGAL
+  FCSR:
+    location: 1
+    description: |
+      The FCSR bit controls access to fcsr for the case when floating-point instructions
+      operate on x registers instead of f registers as specified by the Zfinx and related
```

Comment:
backtick `f` here.

General rule: if it's fixed-font-bold in the spec, it should be backticked in the description fields.